### ISSUE TITLE
Alerts "message" in MetalK8s UI are not always specific enough

### ISSUE THREAD

**Issue by gdemonet** [scality/metalk8s#3127]
**Component**: ui, alerting

**What happened**:

On a development cluster (MetalK8s 2.7.2-dev revision 8d95aa5), NTP was not configured on the hosts, hence Prometheus firing the following alerts:

```javascript
{
  annotations: {
    message: "Clock on 192.168.1.5:9100 is not synchronising. Ensure NTP is configured on this host.",
    runbook_url: "https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-nodeclocknotsynchronising",
    summary: "Clock not synchronising."
  },
  labels: {
    alertname: "NodeClockNotSynchronising",
    container: "node-exporter",
    instance: "192.168.1.5:9100",
    severity: "warning",
    ...more
  },
  ...more
}
```

In MetalK8s UI, on the Monitoring page, I had three rows (one per node), each showing in the Message column: `Clock not synchronising.`.

**What was expected**:

I would have expected to get the full message (`alert.annotations.message`) since there is one in the original alert.
In this case, each row would have shown `Clock on ${nodeIP}:9100 is not synchronising. Ensure NTP is configured on this host.`, making it obvious for me which nodes are affected.

**Steps to reproduce**:

Manually fire any alert with both a `message` and a `summary` annotations, observe that only the `summary` value is shown in MetalK8s UI's Monitoring page.

**Resolution proposal** (optional):

Reorder the priority of fields as defined in #3038 (should be `message` first, then `description`, then `summary`).

So, in both `ui/src/components/VolumeAlertsTab.js` and `ui/src/containers/ClusterMonitoring.js`:

```diff
      return {
        name: alert.labels.alertname,
        severity: alert.labels.severity,
-       message: alert.annotations.description || alert.annotations.summary || alert.annotations.message,
+       message: alert.annotations.message || alert.annotations.description || alert.annotations.summary,
        activeAt: alert.startsAt,
      };
```

**Comment by ChengYanJin** [comment]
Hi @gdemonet,

Thanks for getting us to notice this bug!

I actually realized this issue after we merge #3038, so I fixed it in commit [f06c3a3](https://github.com/scality/metalk8s/pull/3045/commits/f06c3a3d6fb7080cbdae10742e35cec9db40bff2). 
I just reversed the order of `description` and `message`, but since the two fields don't exist at the same time, I think it should be OK.
 ```js      
 message: alert.annotations.description || alert.annotations.message || alert.annotations.summary,
```
The thing we should consider is since I fixed in **dev/2.8**, we may want to backport this fix.
